symforce/slam/imu_preintegration/preintegrated_imu.py

```diff
@@ -0,0 +1,105 @@
+# ----------------------------------------------------------------------------
+# SymForce - Copyright 2022, Skydio, Inc.
+# This source code is under the Apache 2.0 license found in the LICENSE file.
+# ----------------------------------------------------------------------------
+
+import numpy as np
+
+from sym.factors.imu_preintegration_update import imu_preintegration_update
+
+
+class PreintegratedImu:
+    """
+    Numerical Python class for IMU preintegration.
+
+    Usage example:
+        pim = PreintegratedImu(...)
+
+        for meas in imu_measurements:
+            pim.integrate_measurement(meas.accel, meas.gyro, meas.dt, epsilon=epsilon)
+    """
+
+    def __init__(
+        self,
+        accel_cov: np.ndarray,
+        gyro_cov: np.ndarray,
+        accel_bias: np.ndarray,
+        gyro_bias: np.ndarray,
+    ):
+        """
+        Args:
+            accel_cov:
+            gyro_cov:
+            accel_bias:
+            gyro_bias:
+        """
+        # IMU measurement white noise covariance
+        assert accel_cov.shape == (3, 3)
+        self.accel_cov = accel_cov
```

Comment:
suggest do a copy here `self.accel_cov = accel_cov.copy()`, otherwise `accel_bias` could be unintentionally modifed outside.